Extensions/Firebase/C_firebasetools/C_firebasetools.js

```
@@ -0,0 +1,29 @@
+/**
+ * Firebase Tools Collection
+ * @fileoverview
+ * @author arthuro555
+ */
+
+/**
+ * Firebase Event Tools
+ * @namespace
+ */
+gdjs.evtTools.firebase = {};
+
+/**
+ * An array of callbacks to call when the app gets initialized
+ */
+gdjs.evtTools.firebase.onAppCreated = [];
+
+gdjs.registerFirstRuntimeSceneLoadedCallback(function (runtimeScene) {
+  let firebaseConfig;
+  try {
+    firebaseConfig = JSON.parse(
+      runtimeScene.getGame().getExtensionProperty("Firebase", "FirebaseConfig")
+    );
+  } catch {
+    console.error('The Firebase configuration is invalid!');
```

Best practice: don't swallow the error, at least log it:

```
catch(error) {
  console.error('The Firebase configuration is invalid:', error);
}
```